src/components/Account/Login/Login.actions.js

```diff
@@ -119,6 +121,14 @@ export function login({ email, password, activatedData }, type = 'local') {
           if (loginData.settings.speech.rate) {
             dispatch(changeRate(loginData.settings.speech.rate));
           }
+          if (loginData.settings.speech.elevenLabsApiKey) {
```

Comment:
Add the optional changing to avoid unexpected errors 


```suggestion
          if (loginData?.settings?.speech?.elevenLabsApiKey) {
```